releasenotes/notes/asm-detect-attacks-on-the-server.request.cookies-c138bee409617ff0.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    ASM: Detect attacks on Flask request cookies
+  - |
+    ASM: Detect attacks on Django request cookies
+
```

Comment:
Worth doing these in separate PRs?

Once one is reviewed and merged then it should be pretty easy to review and merge the second.

Reply:
Cookies collection in Django has already implemented, but it wasn't verified with tests if Waf detects an attack on these, therefore, It wasn't added in other release notes